.mailmap

```diff
@@ -382,3 +382,10 @@ Ravi charan <ravicharan.vsp@gmail.com>
 Nityananda Gohain <nityanandagohain@gmail.com>
 Nicholas Bollweg <nick.bollweg@gmail.com> <nbollweg@continuum.io>
 Johan Blåbäck <johan_bluecreek@riseup.net> <johan.blaback@cea.fr>
+Adwait Baokar <adwaitbaokar18@gmail.com> abaokar-07 <adwaitbaokar18@gmail.com>
+Rahil Hastu <rahilhastu@gmail.com> rahil hastu <rahilhastu@gmail.com>
+Sidhant Nagpal <sidhantnagpal97@gmail.com> sidhantnagpal <sidhantnagpal97@gmail.com>
+Sidhant Nagpal <sidhantnagpal97@gmail.com> “sidhantnagpal” <sidhantnagpal97@gmail.com>
+Sidhant Nagpal <sidhantnagpal97@gmail.com> Sidhant Nagpal <36465988+sidhantnagpal@users.noreply.github.com>
```

Comment:
Is this file autogenerated?
I would just want `Sidhant Nagpal <sidhantnagpal97@gmail.com>`, just like in AUTHORS.

Reply:
If the version in AUTHORS is the one you want then this file is correct. See the comments at the top of the file.